docs/how-to/add-an-application.mdx

```diff
@@ -76,70 +95,65 @@ Select the [authenticator configuration type](/docs/next/authentication).
     </TabItem>
   </Tabs>
 
-
-
-Click **Submit**. You'll be taken to the app page for the new application. Here, you'll be able to a number of things, such as edit the basic and advanced settings or configure the signup/login experience. 
-
+Click **Submit**. You'll be taken to the app page for the new application. Here, you'll be able to a number of things, such as edit the basic and advanced settings or configure the signup/login experience.
 
 ## API
 
-Before creating an application by API, you must have an [API access token](/docs/next/create-api-token#api). You'll also need to create a Resource Server and an Authenticator Config, which you'll do below. A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. An Authenticator Config defines the invocation type, invoke URL, and trusted origins. 
+Before creating an application by API, you must have an [API access token](/docs/next/create-api-token#api). You'll also need to create a Resource Server and an Authenticator Config, which you'll do below. A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. An Authenticator Config defines the invocation type, invoke URL, and trusted origins.
 
-1. Create a [Resource Server](/docs/next/add-resource-server#api). A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. 
+1. Create a [Resource Server](/docs/next/add-resource-server#api). A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application.
 
   <ResourceServerDescription />
 
   <MultiLanguageCodeBlock
-  curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/resource-servers" \
-  -X POST \
-  -H "Authorization: Bearer $(TOKEN)" \
-  -H "Content-Type: application/json" \
-  -d "{\"resource_server\":{\"display_name\":\"$(APPLICATION_DISPLAY_NAME)\",\"identifier\":\"$(APPLICATION_URI)\",\"scopes\":[\"$(SCOPE)\"]}}"'
-  title="/resource-servers"
+    curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/resource-servers" \
+    -X POST \
+    -H "Authorization: Bearer $(TOKEN)" \
+    -H "Content-Type: application/json" \
+    -d "{\"resource_server\":{\"display_name\":\"$(APPLICATION_DISPLAY_NAME)\",\"identifier\":\"$(APPLICATION_URI)\",\"scopes\":[\"$(SCOPE)\"]}}"'
+    title="/resource-servers"
```

Comment:
nit: `MultiLanguageCodeBlock` works somewhat strangely. When the curl statement is indented the same as the tag, it becomes multi-line. Otherwise, it becomes single-line. It might be worth investigating at some point.

```suggestion
  curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/resource-servers" \
  -X POST \
  -H "Authorization: Bearer $(TOKEN)" \
  -H "Content-Type: application/json" \
  -d "{\"resource_server\":{\"display_name\":\"$(APPLICATION_DISPLAY_NAME)\",\"identifier\":\"$(APPLICATION_URI)\",\"scopes\":[\"$(SCOPE)\"]}}"'
  title="/resource-servers"
```

Reply:
fixed